host_core/mix.exs

```diff
@@ -69,5 +71,10 @@ defmodule HostCore.MixProject do
       {:benchee, "~> 1.0", only: :test},
       {:mock, "~> 0.3.0", only: :test}
     ]
+
+    case :os.type() do
```

Comment:
A TODO here, linked to an issue, explaining we're avoiding bakeware on windows for now, would be nice